packages/crowdsourcing/graphql/resolvers/_mutations/submitInheritanceStory.js

```diff
@@ -0,0 +1,24 @@
+const { sendMail } = require('@orbiting/backend-modules-mail') // holt nur was in {} verlangt wurde
+const { ensureStringLength } = require('@orbiting/backend-modules-utils')
+
+const {
+  CS_INHERITANCE_TO,
+  CS_INHERITANCE_SUBJECT
+} = process.env
+
+module.exports = (_, args, context) => {
+  // const { t } = context
```

Comment:
Remove unused (commented) code

Reply:
Frontend Code available here: https://github.com/orbiting/erbschaft

Reply:
@isabelleSchwab could you add the email verification check? Then this unused code will be used as well ;)